DirectProgramming/DPC++FPGA/Tutorials/Features/mem_channel/sample.json

```diff
@@ -0,0 +1,63 @@
+{
+  "guid": "8EC73D58-B010-4080-9785-B5D9DB13EF3F",
+  "name": "Mem Channels",
+  "categories": ["Toolkit/oneAPI Direct Programming/DPC++ FPGA/Tutorials/Features"],
+  "description": "An Intel® FPGA tutorial demonstrating how to use the mem_channel buffer property and the -Xsno-interleaving flag",
+  "toolchain": ["dpcpp"],
+  "os": ["linux", "windows"],
+  "targetDevice": ["FPGA"],
+  "builder": ["ide", "cmake"],
+  "languages": [{"cpp":{}}],
+  "ciTests": {
+    "linux": [
```

Comment:
For linux tests, we have no long path issues, so please get rid of the change in directory, just work in the local directory.  Note this will also change your cmake commands.